adding initial angular application

noahthomas88 opened the pull request:
This was the initial angular frontend we demoes to Chenoweth, Shows a view of students with their ID name and whether or not they are COVID positive.

Commit:
adding initial angular application

meistejw reviewed (dismissed):
We need to get this code out of a submodule.

DRSisco committed:
Removed Submodule and reference
git rm angular-covid-tracer
Also deleted the submodule reference in .git/config

noahthomas88 committed:
Added Directory
`git add -A` Adds everything that could be added at this point.

DRSisco commented:
Alright, I added a PR into this PR with my de-submodulization and how I went about doing that.
https://github.com/RoseContactTracer/contact-tracer-application/pull/9
Hope that helps.

noahthomas88 committed:
Merge pull request #9 from RoseContactTracer/DRSisco/initial-angular-frontend

De-Submoduled Angular Frontend

meistejw reviewed (approved):
Looks good to me.
